src/coins.cpp

```diff
@@ -268,6 +268,14 @@ double CCoinsViewCache::GetPriority(const CTransaction& tx, int nHeight, CAmount
     return tx.ComputePriority(dResult);
 }
 
+int CCoinsViewCache::GetCoinDepth(const COutPoint& output, int nHeight) const
+{
+    const Coin& coin = AccessCoin(output);
+    if (!coin.IsSpent())
+        return nHeight - coin.nHeight + 1;
```

Comment:
I'd prefer not to duplicate `nHeight - pcoinsTip->GetCoinHeight(output) + 1` everywhere.

Reply:
yeah ok, I'm possibly being over protective but the problem that I see with this is that we don't know inside of the method if the fed height argument is the tip height or not.
Nowadays, we know which functions are calling this and all is good but in some future, maybe someone else inputs another height into the method and it will not be so evident to the reviewer's eyes.

But np.

Reply:
well... in that case a more precise name for the function could be enough. E.g. `GetCoinDeptAtHeight()`.
We can change it in a successive refactoring if we really feel it's worth it.

Reply:
yeah agree, makes much more sense to call it in that way and done.